### ISSUE TITLE
Keyboard navigation does not work for the team dropdown when not signed in.

### ISSUE THREAD

**Issue by berlin-ab** [concourse/concourse#2394]
When trying to log into Concourse and asked to select a team, I would like to be able to filter the dropdown to my team, then use the keyboard cursor keys to select my team, and then be able to press enter to submit my selection. 

Currently, the keyboard cursor keys have no effect on the dropdown selection.

**Comment by jama-pivotal** [comment]
Neat!....except we killed the dropdown for team selection in Concourse 4.0.0. It has Users and won't ask you to choose a team to login to anymore...it just knows!
